packages/elements/examples/nextjs/app/sign-up/[[...sign-up]]/page.tsx

```diff
@@ -227,6 +227,11 @@ export default function SignUpPage() {
             <CustomResendable />
           </SignUp.Strategy>
         </SignUp.Step>
+
+        <SignUp.Step name='restricted'>
+          <H1>Restricted Access</H1>
+          <P>Access to this app is limited, and an invitation is required to sign up.</P>
```

Comment:
@tmilewski we should add a `Back to sign in` button here

Reply:
We can, this is just an internal example app for testing though. I can add this in the follow-on PR.